core/src/main/java/de/danoeh/antennapod/core/util/comparator/FeedItemPubdateComparator.java

```diff
@@ -1,13 +1,17 @@
 package de.danoeh.antennapod.core.util.comparator;
 
+import java.util.Collections;
 import java.util.Comparator;
 
 import de.danoeh.antennapod.core.feed.FeedItem;
 
 /** Compares the pubDate of two FeedItems for sorting*/
 public class FeedItemPubdateComparator implements Comparator<FeedItem> {
 
-	/** Returns a new instance of this comparator in reverse order. 
+	public static final Comparator<FeedItem> descending = new FeedItemPubdateComparator();
+	public static final Comparator<FeedItem> ascending = Collections.reverseOrder(new FeedItemPubdateComparator());
```

Comment:
Are these still needed? If they are needed, they should have a capitalized name, I think.